provisioner/roles/manage_ec2_instances/tasks/addhost_security.yml

```diff
@@ -156,7 +156,7 @@
         ansible_port: "{{ ssh_port }}"
         ansible_ssh_private_key_file: "{{ playbook_dir }}/{{ec2_name_prefix}}/{{ec2_name_prefix}}-private.pem"
         private_ip: "{{item.private_ip_address}}"
-        private_ip2: "{{ item['network_interfaces']|map(attribute='private_ip_address')|list|ipaddr('172.17.0.0/16')|first }}"
```

Comment:
@maxamillion Is this change necessary? We introduced this to make sure that we really get the correct IP address.
AWS cannot be trusted to properly sort the interfaces - sometimes `[network_interfaces[1]` is eth1, sometimes it is eth0.

Reply:
@liquidat yes, that expression causes a python traceback because AWS is doing weird stuff. We can do another map expression if we still need that logic but we're going to have to figure out how to tease out the data we need from the dict returned from the AWS API because for whatever reasons the datastructure we get back from the instance creation is *very* different if we supply our own ENIs or if we ask AWS to create interfaces for us at the time of instance creation.

Reply:
Ah, ok. Thanks.